pilz_robot_programming/test/integrationtests/tst_api_gripper.py

```diff
@@ -33,6 +34,7 @@ class TestAPIGripper(unittest.TestCase):
 
     def setUp(self):
         self.robot = Robot(API_VERSION)
+        self.robot_motion_observer = RobotMotionObserver("gripper")
```

Comment:
You could work with `RobotMotionObserver(group_name = "gripper") to be move expressiv.

Reply:
done.